multimodal/setup.py

```diff
@@ -61,7 +61,7 @@
         "black~=22.0,>=22.3",
         "isort>=5.10",
         "datasets>=2.3.2,<=2.3.2",
-        "onnxruntime-gpu>=1.12.1,<=1.12.1",
+        "onnxruntime-gpu>=1.12.1,<=1.12.1;platform_system!='Darwin'",
```

Comment:
The master branch job failed due to timeout. I wonder whether it's due to the installation of `datasets` and `onnxruntime-gpu`.

Reply:
https://github.com/awslabs/autogluon/pull/2028 The master branch seems to start fail since this PR

Reply:
I see. It's lightning upgrade. Do you think we need to fall back to the previous?